src/redux/reducers/notifications/notifications.js

```diff
@@ -0,0 +1,22 @@
+import {
+    ADD_NOTIFICATION,
+    REMOVE_NOTIFICATION,
+    CLEAR_NOTIFICATIONS
+} from '../../action-types';
+import { generateID } from '../../../functions/generateID';
+
+const addNotification = (notitications, { payload }) => [ ...notitications, { ...payload, id: generateID('notyfication') }];
+const removeNotification = (notifications, { payload }) => {
+    const index = notifications.findIndex(({ id }) => id === payload);
```

Comment:
Please wrap the payload with parseInt: `parseInt(payload, 10)`

Reply:
@karelhala but the id generated by `generateID` is string, isn't it?

Reply:
Oh yeah, that's right. Keep as is.